src/main/java/org/grouplens/grapht/reflect/internal/ClassInstantiator.java

```diff
@@ -153,11 +185,19 @@ public Object instantiate() throws ConstructionException {
                         }
                         throw new ConstructionException(sd, message, e);
                     }
+                    finally {
+                        mdcContextSetterInjectionPoint.finish();
+                    }
                 }
             } else if (d.getInjectionPoint() instanceof NoArgumentInjectionPoint) {
                 // just invoke the method
+                LogContext mdcContextInjectionNoArguments = LogContext.create();
                 Method method = ((NoArgumentInjectionPoint) d.getInjectionPoint()).getMember();
                 try {
+                    mdcContextInjectionNoArguments = LogContext.create();
+                    method = ((NoArgumentInjectionPoint) d.getInjectionPoint()).getMember();
+                    logger.trace("Injection point method with no argument in progress {}", method);
+                    mdcContextInjectionNoArguments.put("org.grouplens.grapht.injectionPoint.noargumentinjectionpoint.method", method.toString());
```

Comment:
All injection points should be put in the 'org.grouplens.grapht.injectionPoint' key.